custom/ilsgateway/api.py

```diff
@@ -148,6 +148,9 @@ def prepare_commtrack_config(self):
         - Sets the proper location types hierarchy on the domain object.
         - Sets a keyword handler for reporting receipts
         """
+        for location_type in LocationType.objects.by_domain(self.domain):
```

Comment:
makes sense. however, this will also delete all locations associated with these types. just confirming that this function only ever gets called once per domain and then it is never called again.